pyiron_atomistics/lammps/interactive.py

```diff
@@ -649,3 +731,14 @@ def interactive_close(self):
                 if "interactive" in h5.list_groups():
                     for key in h5["interactive"].list_nodes():
                         h5["generic/" + key] = h5["interactive/" + key]
+
+
+class _CallBack:
```

Comment:
It's a private class so I don't mind if it's undocumented, *but* if you can think of something useful to say to Fred McFredderson, the MSc student who will join us in five years and wants to fix/extend this class and is looking for some high-level description of intent/promises/etc, it wouldn't hurt.

Reply:
I'm actually waiting for the pyiron meeting today to see what would be the final structure, because creating this class might not be a good idea in the first place anyway, but I promise that I'll add some text before I merge it.